.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - feature/carthage
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: macos-12
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup
+        run: |
+          carthage update --use-xcframeworks
+          pod install
+
+      - name: Run Unit Tests
+        run: xcodebuild -scheme ZincTests -workspace Zinc-ObjC.xcworkspace -destination 'platform=iOS Simulator,name=iPhone 14' -resultBundlePath TestResults test
+    
+      - uses: kishikawakatsumi/xcresulttool@v1
+        with:
+          path: TestResults.xcresult
+        if: success() || failure()
```

Comment:
Why use `success() || failure()` instead of `always()`?

Reply:
For my own learning, why do we need to include this `if` statement in this step?